resource-aggregate/cqrs/eventstore/mongodb/eventstore.go

```diff
@@ -315,13 +339,14 @@ func (s *EventStore) Close(ctx context.Context) error {
 }
 
 // newDBEvent returns a new dbEvent for an eventstore.
-func makeDBEvents(events []eventstore.Event, marshaler MarshalerFunc) ([]bson.M, error) {
+func makeDBEvents(events []eventstore.Event, marshaler MarshalerFunc) (*eventstore.ETagData, []bson.M, error) {
```

Comment:
I guess `makeDBEventsAndGetETag` would be better now

Reply:
fixed